scene/3d/arvr_nodes.cpp

```diff
@@ -127,15 +127,15 @@ Point2 ARVRCamera::unproject_position(const Vector3 &p_pos) const {
 	return res;
 };
 
-Vector3 ARVRCamera::project_position(const Point2 &p_point) const {
+Vector3 ARVRCamera::project_position(const Point2 &p_point, float z_depth) const {
```

Comment:
Parameters should start with `p_`, so even though it's a bit awkward it should be `p_z_depth` here.

Reply:
Ah, thank you. That has been corrected.